src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/migration-to-workflows.mdx

```diff
@@ -0,0 +1,627 @@
+---
+title: Notification migration 
+metaDescription: Migrating notifications from alerts and applied intelligence 
+redirects: 
+---
+The classic notification system for alerts and applied intelligence is being replaced by workflows. This is good news because workflows is an improved, flexible notification system. Workflows help your team learn about potential errors within the greater context of your stack so you can take immediate and efficient action.
+
+What does [this migration](https://discuss.newrelic.com/t/plan-to-upgrade-alert-notification-channels-to-workflows-and-destinations/188205) mean for your team? In our previous notification model, your team would create an alert condition with different thresholds and parameters. If that condition was associated with a specific policy, was violated, and you wanted your team to know about it right away, you would add **notification settings**. Our notification settings would tell New Relic what data to send where.  
+
+Now, we've added workflows. Going forward, instead of configuring classic notification channels and associating them to policies, notification destinations will be created and associated to workflows. Workflows can process data from a range of policies and can use attributes like tags or priority to organize notifications.
+
+## What to expect [#what-to-expect]
+
+The migration from the classic notification system to workflows creates a **destination** for each **classic notification channel** and connects them to **workflows** created for each **policy.** Only policies with channel associations will be migrated.
+
+1. New Relic will automatically migrate accounts from January 9-31 2023.
+2. New Relic can migrate accounts earlier, just reach out to your account team.
+3. Your team can avoid the automated migration by removing channels from policies.
+4. Notifications will not substantially change, specify which values and for which notification channel types.
```

Comment:
> specify which values and for which notification channel types

This was more of a TODO for us.  It might be best to take #4 out, put it into a ### Notification changes section and itemize [the changes](https://github.com/newrelic/docs-website/pull/10619#discussion_r1047620295).

Reply:
done